cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -147,8 +147,7 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     prepareAndTestQuery(
       title = "List indexes",
       text =
-        """Listing all indexes can be done with `SHOW INDEXES`, which will produce a table with the following columns:
-          |include::list-indexes-table-columns.asciidoc[]
+        """include::list-indexes-table-columns.asciidoc[]
```

Comment:
This is still not rendering, might be ok to merge and create card on team docs trello about it, @AlexicaWright will check